lib/components/home/preview_card.dart

```diff
@@ -197,8 +201,8 @@ class _PreviewCardState extends State<PreviewCard> {
     Widget card = MouseRegion(
```

Comment:
I think we could change this so that if we have some files already selected, tapping another (unselected) file will add it to the selection rather than opening the note.
(For this we'd need to conditionally call the `action` parameter from below: `closedBuilder: (context, action) => card`.)